configs/rpt/logger.conf

```diff
@@ -15,6 +15,6 @@ dateformat=%F %T.%3q   ; with milliseconds
 ; Special filename "console" represents the system console
 ;
 ;debug.log => debug
-console => notice,warning,error,dtmf
+console => notice,warning,error,dtmf,debug,verbose
```

Comment:
should we just remove the commented line following this, or should we swap the comment `;` lines and leave the original commented out?